README.md

```diff
@@ -16,3 +16,15 @@ To show it in the sidebar, add the `name` from the repo's `antora.yml` in
 inside the [`icons folder`](./ui/theme/images/icons) matching the name.
 
 Finally, add an entry for it inside the [`model.yml`](./ui/preview/model.yml) file, for UI development purposes.
+
+### Pre-processing
+
+The contents of each repository under `sources` in `playbook.yml` are used as-is
+with no pre-processing. If the repository requires a pre-processing step, it must
+be done in the CI of that repository and the results pushed to a branch that should
+be then specified as the source in `playbook.yml`.
```

Comment:
```suggestion
be done in the CI of that repository and the results pushed to a branch (e.g., `docs`).
This branch should then be specified as the source in `playbook.yml` instead of the
source branch (e.g., `HEAD`).
```